jans-lock/lock-master/service/src/main/java/io/jans/lock/service/util/ServerUtil.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Janssen Project software is available under the Apache License (2004). See http://www.apache.org/licenses/ for full text.
```

Comment:
Please align with this format: https://www.apache.org/licenses/LICENSE-2.0#apply